### DISCUSSION TITLE
Can i Use My Phone number to login, is that mode of authentication possible?

### DISCUSSION THREAD

**Discussion by guru-web3** [Web3Auth/Web3Auth#342] [Help]
Say i want to login with my mobile number, is that mode supported ?

**Comment by himanshuchawla009** [comment]
By default, We don't provide plug-and-play phone number login. But you can add phone number login by setting up Custom Verifier for phone number login. You can set it up on your own server or you can set up using proxy providers like Auth0, firebase or Aws cognito.

Please refer to doc here to know more about setting up custom verifier:-
https://web3auth.io/docs/custom-authentication/verifiers
